app/helpers/html-safe.js

```diff
@@ -0,0 +1,8 @@
+import { helper } from '@ember/component/helper';
+import { htmlSafe as markAsSafe } from '@ember/template';
+
+export function htmlSafe([content] /*, hash*/) {
```

Comment:
Can we use array destructuring on all browsers we support? Or do we compile the JS files somehow?

Reply:
`config/targets.js` tells ember-cli-babel which browsers we support so it knows which features to compile, and I think our listed browsers support array destructuring. I'd have to check UCAndroid.